python/ray/tests/test_client_warnings.py

```diff
@@ -19,33 +17,6 @@ def reset_debug_logs():
 class LoggerSuite(unittest.TestCase):
     """Test client warnings are raised when many tasks are scheduled"""
 
-    def testManyTasksWarning(self):
-        with ray_start_client_server() as ray:
-
-            @ray.remote
-            def f():
-                return 42
-
-            with self.assertWarns(UserWarning) as cm:
-                for _ in range(TASK_WARNING_THRESHOLD + 1):
-                    f.remote()
-            assert f"More than {TASK_WARNING_THRESHOLD} remote tasks have " \
-                "been scheduled." in cm.warning.args[0]
-
-    def testNoWarning(self):
-        with ray_start_client_server() as ray:
-
-            @ray.remote
-            def f():
-                return 42
-
-            with warnings.catch_warnings(record=True) as warn_list:
-                for _ in range(TASK_WARNING_THRESHOLD):
-                    f.remote()
-            assert not any(f"More than {TASK_WARNING_THRESHOLD} remote tasks "
-                           "have been scheduled." in str(w.args[0])
-                           for w in warn_list)
-
```

Comment:
Why are we removing this? this is one of our important usability warnings.

Reply:
CC @ckw017

Reply:
Yes, because it is no longer necessary with async task calls.

Reply:
Wouldn't the overall performance be the same if the network BW is abused?

Reply:
Each task is very small e.g. < 1kb. The issue has been blocking on round trip latency instead of throughput, which is resolved now. And in general, having unnecessary warnings can confuse users and degrade the experience.

Reply:
Should we also remove the logic this is testing?

Reply:
Sounds good to me, avoiding the round-trip cost should definitely be a big help for this case.